src/init.cpp

```diff
@@ -1263,7 +1263,7 @@ static ChainstateLoadResult InitAndLoadChainstate(
         }
         std::tie(status, error) = catch_exceptions([&] { return VerifyLoadedChainstate(chainman, options); });
         if (status == node::ChainstateLoadStatus::SUCCESS) {
-            LogPrintf(" block index %15dms\n", Ticks<std::chrono::milliseconds>(SteadyClock::now() - load_block_index_start_time));
+            LogInfo("Block index %15dms\n", Ticks<std::chrono::milliseconds>(SteadyClock::now() - load_block_index_start_time));
```

Comment:
I keep on forgetting that we can drop the newline now too -_-. All these log lines here after (and including) `"Loading block index..."`  are just kind of bad to be honest. We are definitely doing more here than loading the block index. I'm not even sure if the measured time here provides anything useful given that it varies quite a bit form run to run and we already log timestamps by default. Maybe this should rather just say `"Loaded block index and chainstate"`?

Reply:
Good idea, [done](https://github.com/bitcoin/bitcoin/compare/2ee82dfdeb3777c1d1648daa7e009c420caf5973..60b7bbd3709519c6f2f8023a23ee0a194a5f0eb4)